scripts/distro_script.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -euo pipefail
+
+distro=None
+if [ -f /etc/os-release ]; then
```

Comment:
All this script is doing is checking if certain files exist and based on that decide the distro. Why not just do this in golang itself? Why add this extra layer?

Reply:
ohh nvm I see some extra logic where we're executing some files based on some conditions. I'm not a big fan of doing external calls but since its just a test extension I'm ok with this.